[10.0] [MIG] help_online

lmignon opened the pull request:
Migration of Help_online from 9.0.

lmignon committed:
[MIG] Migrate to 9.0

lmignon committed:
[MIG] Migrate to 10.0

lasley commented on help_online/data/ir_config_parameter_data.xml:
```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='UTF-8' ?>
-<openerp>
+<odoo>
```
`<odoo noupdate="1">` & can remove data tag

lasley commented on help_online/tests/data/help_test_data.xml:
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
   <data>
```
Can remove data tag

lasley commented on help_online/views/help_online_view.xml:
```diff
@@ -1,28 +1,24 @@
+        <field name="context">{"search_default_website":1}</field>
+    </record>
+    
+    <menuitem id="menu_help_main" name="Help Online" groups="help_online_group_writer"/>
```
It would be nice if the web icon was set to the application icon here something like [this](https://github.com/LasLabs/vertical-medical/blob/release/10.0/medical/views/medical_menu.xml#L10)

lasley reviewed (approved):
Thanks @lmignon. Few non-blocking comments, but otherwise awesome module both in terms of code and functionality!

rousseldenis reviewed (approved):
:+1: Code review

njeudy reviewed (approved):
Code review and test

lasley commented:
@lmignon - This is approved, but lacking my non-blocking comments. Were you planning on attending to them before merge?

lmignon committed:
[IMP] help_online: new application icon

lmignon commented:
@lasley Done
![image](https://cloud.githubusercontent.com/assets/544090/23696806/8cf45638-03e5-11e7-9f16-b35680867b1c.png)

lmignon committed:
[FIX] help_online: avoid removal of exported pages
Always use the '__export__' prefix when generating the xmlid of pages exported by the export wizard to avoid autmomatic removal of these pages by Odoo on system update (-u)

lmignon commented:
PLZ don't squash the commits on merge.

lasley commented:
Thanks @lmignon. Merged, no squash!
